app/planets_routes.py

```diff
@@ -0,0 +1,138 @@
+from app import db
+from app.models.planet import Planet
+from app.models.moon import Moon
+from flask import Blueprint, jsonify, abort, make_response, request
+from sqlalchemy import desc, asc
+
+
+# ---------helper functions--------
+def sort_planet_query(planet_query, sort_query, order_query):
+    if not sort_query:
+        sort_query = "name"
+
+    if not order_query:
+        order_query = "asc"
+
+    if order_query == "desc":
+        planet_query = planet_query.order_by(desc(sort_query))
+    else:
+        planet_query = planet_query.order_by(asc(sort_query))
+
+    return planet_query
+
+
+
+def validate_model(cls, model_id):
```

Comment:
Since this function is shared and not specific to the planet routes, I would suggest moving it out of the `planet_routes.py` file.